.vscode/extensions.json

```diff
@@ -0,0 +1,16 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+	// List of extensions which should be recommended for users of this workspace.
+	"recommendations": [
+		"dbaeumer.vscode-eslint",
+		"eamodio.gitlens",
+		"github.vscode-pull-request-github",
+		"octref.vetur"
+	],
```

Comment:
Just a funny VSCode feature that gives us the ability to suggest features to developers who cloned the project.